packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -531,6 +536,21 @@ void main() {
     expect(state.showToolbar(), true);
     await tester.pump();
     expect(find.text('PASTE'), findsOneWidget);
+
+    // Hide the toolbar and clear the text and selection.
```

Comment:
Maybe factor this out as a separate test and mark it as a regression test for https://github.com/flutter/flutter/issues/35998